.github/workflows/ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           - {name: GPU test, options: --use cuda --gpus 1 --debug, log: gpu}
           - {name: 2 GPUs test, options: --use cuda --gpus 2 --debug, log: gpus}
           - {name: OpenMP test, options: --use openmp --omps 1 --ompthreads 2 --debug, log: omp}
```

Comment:
Running the single-OMP tests w/o NUMA memories is by design I assume?

Reply:
We could do the full cartesian product, but I thought testing with more than one socket memory would be sufficient to cover all code paths.